src/views/Points.js

```diff
@@ -18,11 +19,175 @@ import pluralize from 'lib/pluralize';
 
 import View from 'components/View';
 import Blinky from 'components/Blinky';
-import Passport from 'components/Passport';
 import Footer from 'components/Footer';
 import { ForwardButton } from 'components/Buttons';
 import CopiableAddress from 'components/CopiableAddress';
 import NavHeader from 'components/NavHeader';
+import Tabs from 'components/Tabs';
+import PointList from 'components/PointList';
+
+import Locked from './Points/Locked';
+
+function Pending({ incomingPoints, outgoingPoints }) {
```

Comment:
Kinda weird that `Locked` gets its own file, but `Pending` doesn't. It's "only" a 100 lines or so, but that's still nearly a quarter of `Points`. (Guess you'd want to move `Active` into its own file then also, for consistency, but it's so small that it seems hardly worth it.)